### DISCUSSION TITLE
Overlaying an MBTilesImage on other map gives black region

### DISCUSSION THREAD

**Discussion by OlaRune** [pelicanmapping/osgearth#2263] [Support Q&A] [answered]
I'm trying to overlay an MBTilesImage on another map, for example using this earth file:
```
<map>
    <GDALImage name="World">
        <url>../data/world.tif</url>
    </GDALImage>
    <MBTilesImage name="One tile">
	<url>../data/onetile.mbtiles</url>
    </MBTilesImage>
</map>
```

I have uploaded my data here (available for seven days, tell me if I should reupload it): [https://www.transfernow.net/dl/202304119RfrPaXq](https://www.transfernow.net/dl/202304119RfrPaXq)

The data contains one .tif file and one .mbtiles file that I have created using this line:
```
osgearth_conv --in driver GDALImage --in url 64_4_00.tif --out driver MBTilesImage --out filename map.mbtiles --out format jpg
```

When doing this, the MBTilesImage has black borders around it that cover the background map. It differs depending on zoom level, see the attached screenshots.
![screenshot1](https://user-images.githubusercontent.com/25319130/231152100-3acb6f64-c52d-4c68-b0ef-8ffd37ade3d3.PNG)
![screenshot2](https://user-images.githubusercontent.com/25319130/231152108-c51be279-8215-452e-81ef-38e7b7a7c521.PNG)
![screenshot3](https://user-images.githubusercontent.com/25319130/231152111-bc40f334-f4eb-4159-ad08-06e7d412b567.PNG)

How do I remove the black region or make it transparent? I would prefer if this can be done without modifying the .mbtiles file, because I have a much larger file that took ages to process, but if I have to reprocess it I can do that.

**Comment by gwaldron** [comment, marked as answer]
Your mbtiles file appear to be in `jpeg` format, which does not support transparency. You might try regenerating your mbtiles in `png` format.

Another option would be to add a shader to your layer that will look for black pixels and discard them.

**Reply by OlaRune** [reply]
Oh, it was that simple. Thanks!

**Reply by plevy** [reply]
I usually look at this example:
https://github.com/gwaldron/osgearth/blob/master/tests/wms_nexrad.earth
